DEVELOPMENT.md

```diff
@@ -150,7 +150,7 @@ argument name        | meaning                                           | type
 -mode string               | driver run mode                                | string | controller, node, unified | unified
 -nodeid string             | node id                                        | string |              | nodeid
 -registryEndpoint string   | endpoint to connect/listen registry server     | string |              |
-
+-statePath                 | Directory path where to persist the state of the driver running on a node | string | absolute directory path on node | /tmp/pmem-volumes
```

Comment:
This default value is going to be confusing for admins. It's clearly not suitable for persisting state across machine restart. If there is no good default, leave it empty and error out if not set.

Reply:
Done. Now defaults to /var/lib/<drivername>